frontend/src/global/constants.ts

```diff
@@ -47,3 +47,11 @@ export const REGISTRATION_STEPS = [
   ROUTES.Registration.ACCOUNT,
   ROUTES.Registration.CONGRATS
 ];
+
+export enum RedeploymentStatus {
```

Comment:
There should be a way to have this enum auto-generated in the API client?

Reply:
It will be, this is just the frontend implementation with some dummy values until the APIs land (I expected to land earlier this sprint, but there were some delays).

Reply:
Done. The enum is generated from the API spec now.

Reply:
Nice :)